app/containers/FeaturePage/loader.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Asynchronously loads the components for FeaturePage
+ */
+
+import { errorLoading } from 'utils/asyncInjectors';
+
+export default (store, cb) => {
```

Comment:
Thanks for your work! This is awesome, I learned a lot. Just a small question here.

Why do we have to use the callback style? Can't we just return a promise?